runtime/src/main/scala/tailcall/runtime/transcoder/Config2Blueprint.scala

```diff
@@ -300,15 +307,28 @@ object Config2Blueprint {
     }
 
     private def updateModifyField(
+      fieldName: String,
       field: Field,
       bField: Blueprint.FieldDefinition,
+      typeName: String,
```

Comment:
`typeName` is only used for errors. We don't need it because we set it via `trace` at the callsite. See how we fail here https://github.com/tailcallhq/tailcall/pull/219/files#diff-d6df7e22117627a7b52f3ec8ea3c2063effd3dc30928b528a9371ada608e34f7R332